.github/workflows/asan-test.yaml

```diff
@@ -0,0 +1,39 @@
+name: ASAN Tests
+
+on: [push, pull_request]
+
+env:
+  OPEN_UCX_LINK: https://github.com/openucx/ucx
+  OPEN_UCX_BRANCH: master
+  CLANG_VER: 17
+
+jobs:
+  gtest-asan:
+    runs-on: ubuntu-22.04
```

Comment:
Maybe to use ubuntu 24.04? It's also LTS

Reply:
Agree, but I selected 22.04 to be aligned with https://docs.nvidia.com/dgx/dgx-os-6-user-guide/index.html that based on 22.04